src/core/enum-helpers.h

```diff
@@ -48,6 +48,7 @@ std::ostream & operator<<( std::ostream & out, rs2_option option );
 bool try_parse( const std::string & option_name, rs2_option & result );
 
 RS2_ENUM_HELPERS( rs2_stream, STREAM )
+LRS_EXTENSION_API char const * get_abbr_string( rs2_stream );
```

Comment:
Can you give it a meaningfull name?
abbr is not understood

Reply:
It's the standard shorthand for `abbreviated`.

Reply:
I had to open google translate to even understand what abbreviated means.
I would normally use more common words, but it's your call here